Resources/Prototypes/Corvax/Entities/Clothing/Neck/pins.yml

```
@@ -1,3 +1,19 @@
+- type: entity
+  parent: ClothingNeckPinBase
+  id: ClothingNeckAtharaPin
+  name: starry sky pin
+  description: The pin of starry sky.
+  components:
+  - type: Sprite
+    sprite: Corvax/Clothing/Neck/Misc/pins.rsi
+    layers:
+    - state: ath
+  - type: Clothing
+    sprite: Corvax/Clothing/Neck/Misc/pins.rsi
+    clothingVisuals:
+      neck:
+      - state: ath-equipped
```

```suggestion
  - type: Clothing
    sprite: Corvax/Clothing/Neck/Misc/pins.rsi
    equippedPrefix: ath
```
при этом сам стейт должен будет называться `ath-equipped-NECK`
и так же сделай для всех остальных значков